src/main/kotlin/no/nav/infotrygd/kontantstotte/service/InnsynService.kt

```diff
@@ -50,10 +50,9 @@ class InnsynService(
         val stønader = stonadRepository.findByOpphoertVfomEquals("000000")
         logger.info("Fant ${stønader.size} stønader")
 
-        val foedselsnumre = stønader.map { stonad -> stonad.barn.map { barn -> barn.fnr } }
+        val foedselsnumre = stønader.map { stonad -> stonad.barn.map { barn -> barn.fnr.reversert } }
```

Comment:
Jeg tror at å fikse rotfeilen en gang for alle hadde vært bedre: Slik gjøres det i ba-infotrygd

```
@Column(name = "B10_BARN_FNR", columnDefinition = "DECIMAL")
@Convert(converter = ReversedLongFoedselNrConverter::class)
val barnFnr: FoedselsNr,
```

Reply:
Da må man huske å fikse den andre barn.fnr.reversert

Reply:
Kjenner at det er litt vanskelig å teste om jeg har gjort helt riktig...

Reply:
Fort gjort å teste siden dere lagde hent-soekere-og-barn-med-loepende-kontantstoette

Reply:
Har testet. Ser bra ut

Reply:
Vi kan teste sammen

Reply:
Yes, tester gjerne sammen i morgen!